x-pack/plugins/security/server/authentication/api_keys/api_keys.ts

```diff
@@ -29,6 +31,7 @@ export interface ConstructorOptions {
 export interface CreateAPIKeyParams {
   name: string;
   role_descriptors: Record<string, any>;
+  kibana_feature_descriptors?: PutPayloadSchemaType['kibana'];
```

Comment:
I was envisioning these being mutually exclusive options. Specify one or the other, but not both.